### DISCUSSION TITLE
Tagreader with Temperature Sensor

### DISCUSSION THREAD

**Discussion by mlochner90** [adonno/tagreader#155] [Ideas]
Hey, is it possible to add a temperature and humidity sensor to the nfc reader?
I use the Tag Reader in many Rooms and it would be great

**Comment by ghost** [comment]
I also use the the Tagreaders in many rooms (each room of my house). A neat trick that may be cool is turn on  Bluetooth on the ESP so that tagreader could also be a a Bluetooth receiver and then I hacked esp32 Bluetooth sensors could be placed in the same rooms and sensor data collected using the tagreader
